tests/lib/nsxv_test.py

```diff
@@ -861,15 +956,16 @@ def testNsxvStrForDestinationIpv4Mismatch(self):
     # parse the xml
     root = ET.fromstring(sections[1])
 
-    # check destination has only Ipv6
-    destination_address = root.findall('./rule/destinations/destination')
-    self.assertIsNotNone(destination_address)
-    for dest_addr in destination_address:
-      addr_type = dest_addr.find('type').text
-      self.assertEquals(addr_type, 'Ipv6Address')
+    source_addr = root.findall('./rule/sources/source')
+    dest_addr = root.findall('./rule/destinations/destination')
 
-    self.naming.GetNetAddr.assert_has_calls(
-        [mock.call('SOME_HOST'), mock.call('GOOGLE_DNS')])
+    return source_addr, dest_addr
+
+  def verifyAddressType(self, address, type):
```

Comment:
What if you make type a list, and do assert in, that way for the two test cases where it could be either you don't need to replicate the code?